Libplanet.Explorer/Queries/BlockQuery.cs

```diff
@@ -1,4 +1,3 @@
-#nullable disable
```

Comment:
Also making `desc`, `offset`, and `excludeEmptyTxs` could be `NonNullGraphType` here as well.